src/openloco/graphics/gfx.h

```diff
@@ -18,11 +18,55 @@ namespace openloco::gfx
         int16_t pitch;       // 0x0C note: this is actually (pitch - width)
         uint16_t zoom_level; // 0x0E
     };
-
 #pragma pack(pop)
```

Comment:
Needs removing

Reply:
Sorry, must've accidentally reintroduced it during rebase.